app/lib/shared/Core/csaf_2.0_strict.json

```diff
@@ -1,8 +1,7 @@
 {
-  "$id": "https://raw.githubusercontent.com/oasis-tcs/csaf/master/csaf_2.0/json_schema/csaf_json_schema.json#strict",
-  "$schema": "http://json-schema.org/draft-07/schema#",
-  "additionalProperties": false,
-  "definitions": {
+  "$id": "https://docs.oasis-open.org/csaf/csaf/v2.0/csaf_json_schema.json?strict",
```

Comment:
This URL does not resolve for me (`HTTP 404`). Without the `?strict` it does.

Reply:
@bentolor That is correct. However, as the CSAF TC currently doesn't provide that schema which includes the `"additionalProperties": false` this result is intended. It can't be registered without the `?strict` as the "normal" CSAF schema (`app/lib/shared/Core/csaf_2.0.json`) has already that `$id`. Previously, a `#strict` was used to differentiate between the two versions but that is invalid in the `$id`.